lib/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 3.4.1)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++14")
+
+set(LIBS ${CMAKE_SOURCE_DIR}/../MK_DIST/android)
+set(CPP ${CMAKE_SOURCE_DIR}/src/main/cpp)
+
+set(CURL ${LIBS}/curl/7.62.0-0.1.1+2/${ANDROID_ABI})
+set(LIBEVENT ${LIBS}/libevent/2.1.8-0.1.1+3/${ANDROID_ABI})
+set(LIBMAXMINDDB ${LIBS}/libmaxminddb/1.3.2-0.1.1+2/${ANDROID_ABI})
+set(LIBRESSL ${LIBS}/libressl/2.8.2-0.1.1+2/${ANDROID_ABI})
+set(MEASUREMENTKIT ${LIBS}/measurement-kit/0.9.0-0.1.1+1/${ANDROID_ABI})
+
+add_library(libcurl STATIC IMPORTED)
+set_target_properties(libcurl PROPERTIES IMPORTED_LOCATION ${CURL}/lib/libcurl.a)
+
+add_library(libevent_core STATIC IMPORTED)
+set_target_properties(libevent_core PROPERTIES IMPORTED_LOCATION ${LIBEVENT}/lib/libevent_core.a)
+
+add_library(libevent_extra STATIC IMPORTED)
+set_target_properties(libevent_extra PROPERTIES IMPORTED_LOCATION ${LIBEVENT}/lib/libevent_extra.a)
+
+add_library(libevent_openssl STATIC IMPORTED)
+set_target_properties(libevent_openssl PROPERTIES IMPORTED_LOCATION ${LIBEVENT}/lib/libevent_openssl.a)
+
+add_library(libevent_pthreads STATIC IMPORTED)
+set_target_properties(libevent_pthreads PROPERTIES IMPORTED_LOCATION ${LIBEVENT}/lib/libevent_pthreads.a)
+
+add_library(libmaxminddb STATIC IMPORTED)
+set_target_properties(libmaxminddb PROPERTIES IMPORTED_LOCATION ${LIBMAXMINDDB}/lib/libmaxminddb.a)
+
+add_library(libcrypto STATIC IMPORTED)
+set_target_properties(libcrypto PROPERTIES IMPORTED_LOCATION ${LIBRESSL}/lib/libcrypto.a)
+
+add_library(libssl STATIC IMPORTED)
+set_target_properties(libssl PROPERTIES IMPORTED_LOCATION ${LIBRESSL}/lib/libssl.a)
+
+add_library(libmeasurement_kit STATIC IMPORTED)
+set_target_properties(libmeasurement_kit PROPERTIES IMPORTED_LOCATION ${MEASUREMENTKIT}/lib/libmeasurement_kit.a)
+
+add_library(mkall SHARED ${CPP}/ffi.cpp ${CPP}/geoiplookup.cpp ${CPP}/orchestra.cpp ${CPP}/version.cpp)
+
+target_include_directories(mkall PRIVATE ${CPP}/headers ${CURL}/include ${LIBEVENT}/include ${LIBMAXMINDDB}/include ${LIBRESSL}/include ${MEASUREMENTKIT}/include)
+
+target_link_libraries(mkall android libmeasurement_kit libmaxminddb libcurl libevent_openssl libssl libcrypto libevent_extra libevent_core libevent_pthreads log)
```

Comment:
Line 43 and 45 are very long. This makes the diff on GitHub less readable. Do you mind wrapping them?

Reply:
@xanscale this has not been done